types/linkifyjs/index.d.ts

```diff
@@ -94,7 +94,7 @@ export interface LinkifyOptions {
      * NOTE: According to the linkifyjs implementation, `format` can be just a
      * string, but this is not mentioned in the docs, so we exclude it.
```

Comment:
```suggestion
      * string, so we include it (even though this is not mentioned in the docs).
```

But actually maybe move this sentence to just above where `FormatFunc` is defined?